src/components/Selection/Selection.tsx

```diff
@@ -0,0 +1,299 @@
+	) : kind === 'info' ? (
+		<InfoIcon className={cx('&-Icon', `&-Icon-is-${responsiveMode}`)} />
+	) : kind === 'warning' ? (
+		<WarningIcon className={cx('&-Icon', `&-Icon-is-${responsiveMode}`)} />
+	) : null;
+}
+
+interface ISelectionIconProps extends StandardProps {}
+
+const SelectionIcon: FC<ISelectionIconProps> = (): null => null;
+SelectionIcon.peek = {
+    description: `
+        Icon that is displayed within the Selection. Any of the lucid \`*Icon\` components should work.
+    `
+}
+SelectionIcon.displayName = 'Selection.Icon';
+SelectionIcon.propName = 'Icon';
+
+interface ISelectionLabelProps extends StandardProps {}
+
+const SelectionLabel: FC<ISelectionLabelProps> = (): null => null;
+SelectionLabel.peek = {
+    description: `
+        Label for the Selection.
+    `
+}
+SelectionLabel.displayName = 'Selection.Label';
+SelectionLabel.propName = 'Label';
+
+type SelectionKind = 'default' | 'container' | 'success' | 'danger' | 'info' | 'warning'
+
+type SelectionResponsiveMode = 'small' | 'medium' | 'large'
+
+interface ISelectionProps 
+    extends StandardProps,
+        React.DetailedHTMLProps<
+            React.HTMLAttributes<HTMLDivElement>, 
+            HTMLDivElement
+        > {
+    /** Applies an icon and styles for the kind of selection. */
+    kind: SelectionKind,
+
+    /** Apply to the top of a nested sequence of Selection components.
+	 * Adds some spacing for a list of top level Selections with nested Selctions inside each. 
+     * */
+    isTop?: boolean,
+
+    /** Only applies to \`container\` Selection components.
+	 * Fills with a darker gray background.
+	 * Defaults to false. 
+     * */
+    isFilled?: boolean,
+
+    /** Shows or hides the little "x" for a given item. */
+    isRemovable: boolean,
+
+    /** Called when the close button is clicked. */
+    onRemove: ({
+        props,
+        event
+    }: {
+        props: ISelectionProps;
+        event: React.MouseEvent;
+    }) => void,
+
+    /** Gives the selection a background. This is desirable when you only have
+	 * one level of nested selections. 
+     * */
+    hasBackground: boolean,
+    
+    /** Make the content text bold. This is desirable when you only have one
+     * level of nested selections.
+     * */
+    isBold: boolean,
+
+    /** Label of the component. */
+    Label?: React.ReactNode,
+
+    /** Display a custom icon for the selection. Generally you shouldn't need
+	 * this prop since the \`kind\` prop will pick the correct icon for you. 
+     * */
+    Icon?: React.ReactNode,
+
+    /** Adjusts the display of this component. This should typically be driven by
+	 * screen size. Currently \`small\` and \`large\` are explicitly handled by
+	 * this component.
+     * */
+    responsiveMode: SelectionResponsiveMode,
+} 
+
+const defaultProps = {
+    isRemovable: true,
+    onRemove: _.noop,
+    hasBackground: false,
+    isBold: false,
+    kind: 'default' as SelectionKind,
```

Comment:
These are actually better suited using `as const` which tells TS that the value is hard coded and can't be changed. Otherwise you can end up with something bad like this that type checks but is wrong: https://www.typescriptlang.org/play/index.html#code/C4TwDgpgBAYg9nKBeKByOA7CqoB83ADucqA3AFDkCWGwEATgGYCGAxtAAr1xgDOUAb3JQRURggBcsBBQC+lVpl7AodZckHDR4uFNQYEkbFGb94ccrIrlFGdWuAAmKVx78UD0kA